docs/book/advanced-guide/pipelines/scheduling-runs.md

```diff
@@ -0,0 +1,114 @@
+
+# How to schedule a pipeline run
+
+ZenML's scheduling functionality rests on the use of a `Schedule` object that
+you pass in when calling `pipeline.run()`. There are two ways to create a
+schedule with the `Schedule` object, though whether one or both of these are
+supported depends on the specific orchestrator you're using. For example, our
+[Vertex Orchestrator](../../component-gallery/orchestrators/gcloud-vertexai.md) only supports the cron expression method (see below).
+
+You could write a cron expression to describe the pipeline schedule in terms
+that would be comprehensible as [a cron job](https://en.wikipedia.org/wiki/Cron). For example, if you wanted your pipeline to run at 14:05 on Wednesdays, you could use the following:
+
+```python
+from zenml.config.schedule import Schedule
+from zenml.pipelines import pipeline
+
+pipeline_instance = pipeline(step_one=..., step_two=...)
+
+schedule = Schedule(cron_expression="5 14 * * 3")
+pipeline_instance.run(schedule=schedule)
+```
+
+Alternatively, you could manually specify start and end times and the interval between runs. For example, if you wanted your pipeline to run once every 30 minutes starting right now, you could use the following:
+
+```python
+from zenml.config.schedule import Schedule
+from zenml.pipelines import pipeline
+
+pipeline_instance = pipeline(step_one=..., step_two=...)
+
+schedule = Schedule(start_time=datetime.now(), interval_second=1800)
+pipeline_instance.run(schedule=schedule)
+```
+
+You can also optionally specify an `end_time` for the schedule if you want it not
+to run after a certain moment. Similarly, the boolean `catchup` parameter can be
+used to specify whether a recurring run should catch up (i.e. backfill pipeline
+runs) if it is behind schedule, if you paused the schedule, for example.
+
+In the context of scheduled cron or pipeline jobs, backfilling refers to running
+a missed job for a specific period in the past. For example, if a pipeline
+misses a scheduled run at 12:00 PM, backfilling can be used to run the pipeline
+for the 12:00 PM time slot, in order to collect the missing data. This helps
+ensure that the pipeline is up-to-date and that downstream jobs have the
+necessary data to run correctly. Backfilling is a useful technique for catching
+up on missed runs and filling in gaps in scheduled jobs, and can help ensure
+that pipelines and cron schedules are running smoothly. Usually, if your
+pipeline handles backfill internally, you should turn catchup off to avoid
+duplicate backfill. (Here's [a handy
+guide](https://medium.com/nerd-for-tech/airflow-catchup-backfill-demystified-355def1b6f92)
+in the context of Airflow.) Note that the `catchup` parameter enabling
```

Comment:
I think the note should be in a `{% hint style="warning" %}{% endhint %}` in the next line